rpcclient/roundrobin.go

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) 2015-2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package rpcclient
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/btcsuite/websocket"
+)
+
+var (
+	// ErrNoConnAvailable indicates no Conn is available for pick().
+	ErrNoConnAvailable = errors.New("no Connection is available")
```

Comment:
The word connection shouldn't be capitalized.  It is not good practice to refer to the actual struct names in error messages.